Model_rot.f90

```diff
@@ -368,7 +368,6 @@ end subroutine get_parameters
 
  !Allocate the array sizes
  subroutine allocate_vars
-   ig(1)=ix ; ig(2)=jx ; ig(3)=kx   
```

Comment:
Because I am allocating values to `ig` and the individual `_x` variables in `MPI_rot.f90`, there is no need to do it here.